easybuild/easyconfigs/u/UCX/UCX-1.10.0-dynamic_modules.patch

```diff
@@ -0,0 +1,27 @@
+# UCX is hardcoded to only load the modules it is compiled with.
+# This patch adds an optional EB_xxx_MODULES that can override this list, allowing new modules to be loaded later.
```

Comment:
@Micket Comment says `EB_xxx`, while it's actually `EC_UCX_xxx`, we should fix that to avoid confusion?

Reply:
fixed in #12984